ayaneo-platform.c

```diff
@@ -568,8 +699,14 @@ static int ayaneo_platform_probe(struct platform_device *pdev)
         ret = PTR_ERR_OR_ZERO(match);
         if (ret)
                 return ret;
+
+        ret = devm_device_add_groups(dev, ayaneo_led_mc_groups);
```

Comment:
`dev` here adds the attribute to the platform device. During the init we have access to the led_mc dev via ` struct led_classdev *led_cdev = &ayaneo_led_mc.led_cdev;`. If we use this instead, the attribute will be accessible through  `sys/class/leds/multicolor:chassis` with all the other attributes instead of ` /sys/class/leds/multicolor:chassis/device` with the platform attributes. As this is specifically related to the suspend mode of the led_mc I think it makes more sense to have it attached to the led device.

Reply:
I've moved the attribute to the led device as suggested; definitely better that way.